training_step/bert/component_template.yaml

```diff
@@ -0,0 +1,31 @@
+name: train_pytorch_cnn
```

Comment:
Please change the name to reflect the actual bert example

Reply:
The name changes have been done